### PR TITLE
Add `cloudwatchevents` service prow jobs and labels

### PR DESCRIPTION
**Pull request by A-Hilaly**
Add `cloudwatchevents` service prow jobs and labels

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jaypipes — APPROVED** [review]
ty @A-Hilaly!

File: prow/jobs/jobs.yaml [hunk 1]
```diff
@@ -210,7 +210,7 @@ postsubmits:
       base_ref: main
       workdir: false
     - org: aws-controllers-k8s
-      repo: cloudwatch-controller
+      repo: cloudwatchevents-controller
```
Comment (jaypipes) [inline prow/jobs/jobs.yaml:213]: Oh, this needs to be cloudwatchlogs, not cloudwatchevents.

**Review by jaypipes — CHANGES_REQUESTED** [review]

File: prow/jobs/jobs.yaml (hunk 1 above)
Comment (A-Hilaly) [reply prow/jobs/jobs.yaml:213]: we have cloudwatchlogs right under it :)

Comment (A-Hilaly) [reply prow/jobs/jobs.yaml:219]: Fixed the PR title/description

**Review by jaypipes — APPROVED** [review]

**Pull request merged**